### PR TITLE
add support disk io device name to json output

### PR DESCRIPTION
**Pull request by andyxning**
This PR will update cAdvisor dep to support getting device name when [heapster scrape disk io metrics](https://github.com/kubernetes/heapster/pull/1450).

**Release note**:
```release-note
None
```

/cc @dchen1107 @dashpole @vishh

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1311,213 +1311,213 @@
 		},
 		{
 			"ImportPath": "github.com/google/cadvisor/accelerators",
-			"Comment": "v0.28.0",
-			"Rev": "3d2e7fcfa396ab4f193c7720b6a6dc3133b6f50a"
+			"Comment": "v0.28.0-14-g5231853",
```
Comment (dashpole) [inline Godeps/Godeps.json]: please wait for me to cut the next release, which is scheduled for today.   I prefer only vendoring tagged cadvisor versions.  Thanks

Comment (andyxning) [reply Godeps/Godeps.json]: I agree with `tagged cadvisor versions`. But it maybe too late for now to tag a new cadvisor release. :(

This is needed for adding disk io metrics for heapster. I will fix the PR and please take a look at this.

**Comment by brendandburns** [comment]
Need to fix the cross-compile build. otherwise lgtm.

**Comment by andyxning** [comment]
@brendandburns All CI are green. PTAL.

**Comment by andyxning** [comment]
Ping @dashpole

**Comment by andyxning** [comment]
Since @dashpole has bumped cAdvisor dep to 0.28.2. The motivation for this PR has been accomplished by #56109.

**Pull request closed without merging**